x-pack/plugins/security_solution/public/cases/containers/types.ts

```diff
@@ -13,6 +13,7 @@ export interface Comment {
   createdAt: string;
   createdBy: ElasticUser;
   comment: string;
+  type: 'user' | 'alert';
```

Comment:
maybe to avoid all the `as const` casting, we should create an `enum` for this value?
```
enum userType {
  user = 'user',
  alert = 'alert',
}
```